scripts/upload_config.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -e
+trap 'echo "ERROR: ${BASH_SOURCE}:${LINENO} ${BASH_COMMAND}"' ERR
```

Comment:
This will display the line in this script from which any error originates, helpful even when a line makes an error but does not emit stderr for some reason.